my_function.py

```
@@ -2,8 +2,8 @@ def add_numbers(a, b):
     """
     This function adds two numbers.
     """
-    sum = a + b
-    return sum
+    total = a + b  # Renamed 'sum' to 'total'
```

Ruff (0.8.2)

Local variable `total` is assigned to but never used

Remove assignment to unused variable `total`

(F841)